types/sinon/index.d.ts

```diff
@@ -1482,6 +1478,12 @@ declare namespace Sinon {
     type SinonStubbedMember<T> = T extends (...args: infer TArgs) => infer TReturnValue ? SinonStub<TArgs, TReturnValue>
         : T;
 
+    /**
+     * Replaces a method type with a Sinon stub.
+     */
+    type SinonStubbedMethod<T> = T extends (...args: infer TArgs) => infer TReturnValue ? SinonStub<TArgs, TReturnValue>
```

Comment:
maybe lets call it `SinonStubbedFunction` actually since it could be used for any function, not only methods

then update the jsdoc too ("Replaces a function type ...")

Reply:
Done